src/clinical/donor-repo.ts

```diff
@@ -48,6 +48,10 @@ export interface DonorRepository {
     programId: string,
     projections?: Partial<Record<DONOR_FIELDS, number>>,
   ): Promise<DeepReadonly<Donor[]>>;
+  findByProgramIdOmitMongoDocId(
```

Comment:
can we instead use the same previous method with optional arguments to return lean object and use the new method you created as private method ?

Reply:
hmm, I don't see why not